x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/planner/ExpressionTranslators.java

```diff
@@ -78,37 +77,6 @@ public final class ExpressionTranslators {
     public static final String DATE_FORMAT = "strict_date_optional_time_nanos";
     public static final String TIME_FORMAT = "strict_hour_minute_second_fraction";
 
-    public static final List<ExpressionTranslator<?>> QUERY_TRANSLATORS = List.of(
-        new BinaryComparisons(),
-        new Ranges(),
-        new BinaryLogic(),
-        new IsNulls(),
-        new IsNotNulls(),
-        new Nots(),
-        new Likes(),
-        new InComparisons(),
-        new StringQueries(),
-        new Matches(),
-        new MultiMatches(),
-        new Scalars()
-    );
-
-    public static Query toQuery(Expression e) {
```

Comment:
No more accidentally calling into this `toQuery`, rather than `esql`'s one! That had caused bugs in the past.